templates/project_profile_backed_up.html

```diff
@@ -1,8 +1,4 @@
-{% extends "bootstrap/base.html" %}
```

Comment:
this template doesn't seem to be used by app.py. You should delete it as part of this pull request.